tango/integrations/torch/train.py

```diff
@@ -295,18 +294,11 @@ def _train(
     is_distributed: bool = False,
     devices: Optional[List[int]] = None,
     distributed_port: str = "54761",
-    include_package: Optional[List[str]] = None,
     val_metric_name: str = "loss",
     minimize_val_metric: bool = True,
     aggregate_val_metric: bool = True,
     callbacks: Optional[List[Lazy[TrainCallback]]] = None,
 ) -> Optional[Model]:
-    if include_package:
```

Comment:
Tango already includes packages, no? The `Executor` does that before anything runs? Do we need to do it again?

Reply:
This is actually important because when we spawn subprocesses (like during distributed training), those processes need to re-import all of the `include_package` modules, or they won't be able to instantiate things.